algorithms/java/src/org/openda/algorithms/kalmanFilter/SteadyStateFilter.java

```diff
@@ -384,8 +384,7 @@ private void compensateForMissingObservationsWithHK(double[] obsTimeOffsets, IVe
 			Matrix inverseIMinusM2 = iMinusM2.inverse();
 			Matrix m1DAvailable = this.m1.mult(this.dAvailable);
 			Matrix dMissing = inverseIMinusM2.mult(m1DAvailable);
-		/* Code used for debugging when necessary
-		System.out.println("M1: " + m1);
+		/*System.out.println("M1: " + m1);
```

Comment:
Has this been switched off correctly. M1 is commented and M2 not? perhaps a bloc if (DEBUG) {
}
around the prints or use the logger for the output?